src/components/custom/_signup-affiliate-details.js

```diff
@@ -47,28 +49,24 @@ const InputGroup = styled.div`
     }
 `
 const SignupButton = styled(Button)`
-    width: 81px;
-    font-size: 14px;
     margin-top: 40px;
     @media ${device.mobileL} {
         margin-top: 32px;
     }
-    @media ${device.mobileL} {
-        font-size: 12px;
-    }
 `
 const StyledLinkText = styled(LinkText)`
-    font-size: '14px';
+    font-size: 14px;
     @media ${device.mobileL} {
         font-size: 12px;
     }
 `
 const DropdownSearchWrapper = styled.div`
     margin-bottom: -16px;
 `
-const StyledLocalizedLink = styled(LocalizedLink)`
+const StyledLink = styled(LocalizedLink)`
     color: var(--color-red);
     text-decoration: none;
+    line-height: 18px;
```

Comment:
I mean to use `StyledLink` instead of `LocalizedLink`.

Reply:
Replaced, and updated links in agreement labels.